students/rod_musser/lesson03/mailroom.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+import sys
+
+donors = [
+    ['Carmelo Anthony', 1, 50],
+    ['Damien Lillard', 100.50, 99, 10000],
+    ['CJ McCollum', 24000, 70, 100, 5, 300],
+    ['Hassan Whiteside', 10000],
+    ['Terry Stotts', 500, 500, 100, 100, 100],
+]
+
+welcome_prompt = "\n".join(("Welcome to the Local Charity Mail Room System",
+                  "Please choose from the following options:",
+                  "1 - Send a Thank You",
+                  "2 - Create a Report",
+                  "3 - Quit",
+                  ">>> "))
+
+
+def main():
+    while True:
+        response = input(welcome_prompt)
+        if response == "1":
+            send_thank_you()
+        elif response == "2":
+            create_report()
+        elif response == "3":
+            exit_program()
+        else:
+            print("Not a valid option!")
+
+
+def list_donors():
+    newLine = '\n'
```

Comment:
```suggestion
    new_line = '\n'
```

variable names should be snake case